block/block.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "team51/colophon",
+  "version": "0.1.0",
+  "title": "Colophon",
+  "category": "common",
+  "icon": "editor-textcolor",
+  "description": "A block for displaying a colophon.",
+  "supports": {
+    "html": false,
+    "align": true,
+    "typography": {
+      "fontSize": true,
+      "lineHeight": true,
+      "__experimentalFontFamily": true,
+      "__experimentalFontStyle": true,
+      "__experimentalFontWeight": true,
+      "__experimentalLetterSpacing": true,
+      "__experimentalTextTransform": true,
+      "__experimentalDefaultControls": {
+        "fontSize": true
+      }
```

Comment:
I assume these are all managed via the block attributes, which is why we don't have styles implementing them. 👍

Reply:
So, should we keep or remove them?